code/modules/assembly/doorcontrol.dm

```diff
@@ -143,6 +143,6 @@
 	cooldown = TRUE
 	for (var/obj/structure/bodycontainer/crematorium/C in GLOB.crematoriums)
 		if (C.id == id)
-			C.cremate(usr)
+			C.toggle_cremation(usr)
```

Comment:
this doesn't work with the signature you made for toggle_cremation

Reply:
will fix